src/Symfony/Component/HttpClient/Internal/AmpListener.php

```diff
@@ -26,13 +26,15 @@
 class AmpListener implements EventListener
 {
     private array $info;
-    private array $pinSha256;
-    private \Closure $onProgress;
     /** @var resource|null */
     private $handle;
 
-    public function __construct(array &$info, array $pinSha256, \Closure $onProgress, &$handle)
-    {
+    public function __construct(
+        array &$info,
+        private array $pinSha256,
+        private \Closure $onProgress,
+        &$handle,
```

Comment:
```suggestion
        private &$handle,
```

Reply:
I wasn't sure about this because of the reference

Reply:
see https://3v4l.org/sQ1de

Reply:
Thanks! done